GrindingCity.Domain/Entities/Building/BuildingEntity.cs

```diff
@@ -1,8 +1,11 @@
-﻿namespace GrindingCity.Domain.Entities.Building
+﻿using GrindingCity.Domain.Interfaces.Entities;
+
+namespace GrindingCity.Domain.Entities.Building
 {
     public class BuildingEntity
     {
         public Guid Id { get; set; }
-        public string Name { get; set; }
+        public string Name { get; set; } = default!;
+        public IList<IResource> Resources { get; set; } = default!;
```

Comment:
Really often you gonna see collection initialization there instead of null suppression ( `new List<IResource>();`)